app/controllers/controllers.js

```diff
@@ -0,0 +1,17 @@
+// need to require in functions in models
+const { fetchTopicsData } = require("../models/models");
+
+// Controllers:
+const getTopics = (request, response, next) => {
+  fetchTopicsData()
+    .then((topicsData) => {
+      if (!topicsData) {
+        return response.status(404).send({ error: "Topics not found" });
```

Comment:
Nice thinking about error handling, but instead of sending a response status here, a Promise reject would be better and then pass it onto a error handler.